src/test/java/com/internship/frejaeidjmeterplugin/jmeter/frejaRequests/SignServiceTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.internship.frejaeidjmeterplugin.jmeter.frejaRequests;
+
+import com.verisec.frejaeid.client.beans.sign.get.SignResult;
+import com.verisec.frejaeid.client.enums.MinRegistrationLevel;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientInternalException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientPollingException;
+import java.time.Duration;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ *
+ * @author User
+ */
+public class SignServiceTest {
+    
+    private static SignService instance;
+    
+    public SignServiceTest() {
+    }
+    
+    @BeforeAll
+    public static void setUpClass() {
+        System.out.println("Class setup");
+        try {
+            instance = new SignService();
+        } catch (FrejaEidClientInternalException ex) {
+            Logger.getLogger(SignServiceTest.class.getName()).log(Level.SEVERE, null, ex);
+            fail("Bad init");
+        }
+        System.out.println("Done");
+    }
+    
+    @AfterAll
+    public static void tearDownClass() {
+        System.out.println("Class dissolution");
+    }
+    
+    @BeforeEach
+    public void setUp() {
+        System.out.println("pre-test setup");
+    }
+    
+    @AfterEach
+    public void tearDown() {
+        System.out.println("post-test cleanup");
+    }
+
+    /**
+     * Test of initiateSignRequest method, of class SignService.
+     */
+    @Test
+    public void testInitiateSignRequest() throws Exception {
```

Comment:
Rename test to describe what it is testing - in this case it is sucessfull case for init sign.
Do this with all test cases.